koji_containerbuild/cli.py

```diff
@@ -94,11 +101,12 @@ def handle_container_build(options, session, args):
                       help=_("Git branch"))
     parser.add_option("--channel-override",
                       help=_("Use a non-standard channel [default: %default]"),
-                      default=DEFAULT_CHANNEL)<
-    parser.add_option("--release",
-                      help=_("Set release value"))
-    parser.add_option("--koji-parent-build",
-                      help=_("Overwrite parent image with image from koji build"))
+                      default=DEFAULT_CHANNEL)
```

Comment:
Think the 'cleanup' commit had a typo that was fixed in a later commit in this PR.

Reply:
Sorry, fixed in this rebase.